app/graph/types/source_type.rb

```diff
@@ -34,4 +34,14 @@ def medias
 
   field :medias_count, GraphQL::Types::Int, null: true
   field :collaborators, UserType.connection_type, null: true
+
+  def image
+    super_admin? ? "#{CheckConfig.get('checkdesk_base_url')}/images/user.png" : object.image
+  end
+
+  private
+
+  def super_admin?
+    object.user.is_admin && !object.user.is_member_of?(Team.current)
```

Comment:
it's better to use `&`  to avoid raising error for null user 
i.e ` object.user&.is_admin && !object.user&.is_member_of?(Team.current)`

Reply:
@melsawy I realized I added the safe operator in `user_type`, so I added it to `source_type` as well.